### DISCUSSION TITLE
Integrate existing share

### DISCUSSION THREAD

**Discussion by K3vb3rt** [nextcloud/all-in-one#5623] [Questions] [answered]
I am currently installing the Docker in my Unraid. Now I would like to integrate an existing share “Username” into Nextcloud WITHOUT it being listed in “External storage”.

The data should simply be displayed in Nextcloud as if it had been uploaded there.

How can this be implemented? Is a mapping to the usershare sufficient or does something special have to be changed, e.g. permissions?

**Comment by szaimen** [comment, marked as answer]
Hi, please ask this at https://help.nextcloud.com. Thanks! :)
